readme.md

```diff
@@ -80,6 +80,8 @@ Once the database configuration is set, you can run `php artisan migrate` follow
 
 **You MUST set an App Key.**
 
+You may also need to run `find storage/* -type d -exec chmod 775 {} \;` to grant write permissions on your storage folder
```

Comment:
Yes, this is easier - the line I provided was just copied from a shell script of mine. Further, I actually had to do 777 instead of 775 to get it fully working